sphinx/templates/latex/latex.tex_t

```diff
@@ -25,16 +25,6 @@
 \usepackage<%= sphinxpkgoptions %>{sphinx}
 <%= sphinxsetup %>
 <%= geometry %>
-\usepackage{multirow}
-\let\originalmutirow\multirow\protected\def\multirow{%
-  \sphinxdeprecationwarning{\multirow}{1.6}{1.7}
-  {Sphinx does not use package multirow. Its loading will be removed at 1.7.}%
-  \originalmultirow}%
-\usepackage{eqparbox}
-\let\originaleqparbox\eqparbox\protected\def\eqparbox{%
-  \sphinxdeprecationwarning{\eqparbox}{1.6}{1.7}
-  {Sphinx does not use package eqparbox. Its loading will be removed at 1.7.}%
-  \originaleqparbox}%
```

Comment:
I think it is better to emit deprecation warnings only if "easily" possible.
It's okay to announce deprecation through "Incompatible changes" section of CHANGES file.